plugins/importexport/native/filter/NativeXmlSubmissionFileFilter.inc.php

```diff
@@ -197,10 +197,10 @@ function handleRevisionElement($node, $stageId, $fileId) {
 					$submissionFile->setUserGroupId($userGroup->getId());
```

Comment:
Then this is maybe not needed, because the user group will be set after the while loop however, because the $lastuserGroup will exist, correct?